uk.co.compucorp.civicrm.tasksassignments/tasksassignments.php

```diff
@@ -166,6 +166,8 @@ function tasksassignments_civicrm_alterAPIPermissions($entity, $action, &$params
   if (in_array($entity, $entitiesToAvoidPermissions)) {
     $params['check_permissions'] = false;
   }
+
+  $permissions['setting']['get'] = ['access AJAX API'];
```

Comment:
Access to which kind of settings does this permission provide?

Reply:
Access to make ajax call to `Setting.get()` form T&A.

Reply:
Yeah, I understand, but what actually does it allow users to access, on top of the data about max size limit? Could you please provide a link to the documentation that described which data is accessed once you grant that permission?

Reply:
There is no such doc or page that describes the `Settings.get`. Here is what id does though:
> civicrm allows extensions to create many different settings and also has some of its own. All of them are stored in the civicrm_setting table in the database. This API basically returns everything that is in this table

Thanks @davialexandre for making it clear.